mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala

```diff
@@ -135,13 +135,13 @@ abstract class MLWriter extends BaseReadWrite with Logging {
 }
 
 /**
- * Trait for classes that provide [[MLWriter]].
+ * Trait for classes that provide `MLWriter`.
```

Comment:
@jkbradley, Yes, it seems fine for both.

I think I changed them given `sealed trait BaseReadWrite`...

```
[warn] .../spark/mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala:143: Could not find any member to link for "MLWriter".
[warn]   /**
[warn]   ^
```

and I think I swept them here as they looked identical. I am okay to revive identified ones back.

My (maybe nitpicking) concern is, we should be able to identify the cases explicitly which I guess I and many guys failed and also what to do in each case. Otherwise, I think we should prefer backquotes because even if reviving links might work for now for some cases, we could easily introduce other breaks again when we change the codes.